setup.py

```diff
@@ -15,7 +15,7 @@ def __str__(self):
                    libraries=['k4a'])
 
 setup(name='pyk4a',
-      version='0.3',
+      version='0.4',
```

Comment:
We will update it with the PyPi release.

Reply:
will remove - was more for my own testing to see versions updates